agent_identity/src/state.rs

```diff
@@ -78,74 +113,191 @@ pub const VERIFIABLE_PRESENTATION_SERVICE_ID: &str = "linked-verifiable-presenta
 pub async fn initialize(state: &IdentityState) {
     info!("Initializing ...");
 
-    let enable_did_web = config()
+    // Only consider updateable DID methods.
```

Comment:
Let's discuss if we can find a better word for "updateable". I think it reads as if the DID method itself was updateable. Would the classic "mutable" vs. "immutable" fit? So it reads

```
// Only consider DID methods with mutable documents
```

Reply:
Updated the DID method traits (naming) to [DID traits v0.8.0](https://github.com/decentralized-identity/did-traits/blob/v0.8.0/schemas/v0.8.0/traits.json).